.goreleaser.yml

```diff
@@ -0,0 +1,116 @@
+builds:
+  - main: ./cmd/ttn-lw-stack
+    binary: ttn-lw-stack
```

Comment:
If this is built into the workdir (lorawan-stack) it should be added to gitignore. Same for cli below.

Reply:
It's not - `goreleaser` creates a `dist` directory, where all the artifacts are placed.

Reply:
Then we probably need to check the Dockerfile, because https://github.com/TheThingsNetwork/lorawan-stack/pull/109/files#diff-3254677a7917c6c01f55212f86c57fbfR7 copies it from the workdir. We should also probably document how `docker build` needs to be executed since you removed the `make docker` target.

Reply:
Haha, I initially wrote this in https://github.com/TheThingsNetwork/lorawan-stack/pull/109#discussion_r255435612 but then deleted

For `docker`, `goreleaser` creates a temporary directory and copies the required binaries and any `extra_files` specified to the root of that temporary directory.
You can verify that binaries are there by looking/testing `rvolosatovs/lorawan-stack` Docker image(https://cloud.docker.com/u/rvolosatovs/repository/docker/rvolosatovs/lorawan-stack)